presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -160,6 +163,20 @@ public StorageManagerConfig setCompactionInterval(Duration compactionInterval)
         return this;
     }
 
+    @MinDuration("1s")
+    public Duration getOrganizationInterval()
+    {
+        return organizationInterval;
+    }
+
+    @Config("storage.organization-interval")
+    @ConfigDescription("How often to check if local shards are organized well")
```

Comment:
This phrasing is a bit odd. How about "How often to check for local shards that need organization"